autoload/db/adapter/bigquery.vim

```diff
@@ -0,0 +1,81 @@
+if !executable('bq')
+  echom 'bq command is not available.'
+  let g:autoloaded_db_bigquery = 0
+endif
```

Comment:
No other adapter does this; drop it. If you're unhappy with the experience when the db command is missing, that should be fixed in a more general way that works for all adapters.

Reply:
Fair point!